### PR TITLE
RSDK-8900 Do not use networking logger for net appender

### PR DESCRIPTION
**Pull request by benjirewis**
RSDK-8900

Confirmed this change means `rdk.networking` logs appear in app.

### CODE REVIEW

File: web/server/entrypoint.go [hunk 1]
```diff
@@ -150,7 +150,7 @@ func RunServer(ctx context.Context, args []string, _ logging.Logger) (err error)
 				ID:         cfgFromDisk.Cloud.ID,
 				Secret:     cfgFromDisk.Cloud.Secret,
 			},
-			nil, false, logger.Sublogger("networking").Sublogger("netlogger"),
+			nil, false, logger.Sublogger("netlogger"),
```
Comment (dgottlieb) [inline web/server/entrypoint.go:153]: Not a mandate, but `logging.NewLogger("rdk.networking.netlogger")` is also suitable here if we wanted to keep the "name".

**Review by dgottlieb — APPROVED** [review]
Not for this PR (unless there's something obviously useful/correct to add) -- should we be trying to invariant that loggers (just rdk.networking? all loggers?) "always" have a net appender?

Just wondering how we'd learn if we re-introduced this bug by touching code that happens prior to the net appender getting installed.

File: web/server/entrypoint.go (hunk 1 above)
Comment (benjirewis) [reply web/server/entrypoint.go:153]: Ah good point; did `logger.Sublogger("networking.netlogger")` instead. Same idea, right?

**Comment by benjirewis** [comment]
> should we be trying to invariant that loggers (just rdk.networking? all loggers?) "always" have a net appender?

If there were an invariant I would say that after the net appender has been created, all loggers should be using it... Filed https://viam.atlassian.net/browse/RSDK-8902 to try to handle that.

File: web/server/entrypoint.go (hunk 1 above)
Comment (dgottlieb) [reply web/server/entrypoint.go]: I think that's obscuring the intent here. Use `logging.NewLogger` is clear that:
* We're not going into the registry
* We're not "borrowing" anything from the `logger` state.

Also, I think `logger.Sublogger("networking.netlogger")` is relying on an implementation detail. Specifically, `Sublogger` could be implemented as:
* For each "dot-delimited piece" in the sublogger name
  * Get or create that piece

Meaning we'd turn `logger.Sublogger("networking.netlogger")` internally into two calls:
```
logger = logger.Sublogger("networking")
logger = logger.Sublogger("netlogger")
return logger
```
Which would reintroduce the bug.

Comment (benjirewis) [reply web/server/entrypoint.go]: Yepp you're totally right thanks for spelling that out for me 😮‍💨 . Opted for `logging.NewLogger("rdk.networking.netlogger")` as you originally suggested.

**Review by dgottlieb — COMMENTED** [review]
still lgtm!

oh it was merged!

**Comment by benjirewis** [comment]
Oop sorry about that.

**Pull request merged**